src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        final int ballCount = 3;
+        for (int i = 0; i < ballCount; i++) {
+            System.out.println(lottery.getRandomBall().toString());
```

Comment:
```suggestion
            System.out.println(lottery.getRandomBall());
```
`.toString()` is redundant here